dispatcher/DAQController.py

```diff
@@ -246,7 +247,7 @@ def control_detector(self, command, detector, force=False):
             if command == 'start' and self.mongo.insert_run_doc(detector):
                 # db having a moment
                 return
-            if (command == 'stop' and ls[detector]['number'] != NO_NEW_RUN and
+            if (command == 'stop' and ls[detector]['number'] != -1 and
```

Comment:
particular reason why you want to revert to a magic -1?

Reply:
Import errors, you can't do `from .MongoConnect import NO_NEW_RUN`. It was much faster to go back to the tried and true -1. Also, -1 here doesn't mean `NO_NEW_RUN` it means no active run.